### ISSUE TITLE
is this still up?

### ISSUE THREAD

**Issue by mensfeld** [gcao/aspect4r#39]
Maybe would be good to remove it since there's an aspector? It can be confusing when you see two similar repos

**Comment by mensfeld** [comment]
Ping ;)

**Comment by gcao** [comment]
Yeah. I was doing some experiment in this repo a couple months back.

**Comment by rennhak** [comment]
Agree with @mensfeld here. Got initially confused by this as well.

One thing I'm for instance still not sure about is if Aspector is a 'spiritual' successor of Aspect4r or in what relation these projects stand (fork? independent?). A hint in the README file at least would be helpful.

Thank you.
